crates/core/src/time.rs

```diff
@@ -206,6 +206,16 @@ impl DateTimeUtc {
     pub fn next_second(&self) -> Self {
         *self + DurationSecs(1)
     }
+
+    /// Returns the number of seconds in between two `DateTimeUtc` instances.
+    /// Ass
```

Comment:
```suggestion
```

Reply:
fixed in 6ce70ce70223f7e650fe4412373c23bd8bc6e9fb